Improve style nit check for space after a comma

nerith opened the pull request:
Fixes #7345.

<!-- Reviewable:start -->

[<img src="https://reviewable.io/review_button.png" height=40 alt="Review on Reviewable"/>](https://reviewable.io/reviews/servo/servo/7499)

<!-- Reviewable:end -->

nerith committed:
Improve style nit check for space after a comma

Only a small number of things that followed a comma were flagged.
This improves the thoroughness of that check.

Fixes #7345.

highfive commented:
<img src="http://www.joshmatthews.net/warning.svg" alt="warning" height=20> **Warning** <img src="http://www.joshmatthews.net/warning.svg" alt="warning" height=20>

* These commits modify layout code, but no reftests are modified. Please consider adding a reftest!

metajack commented:
Looks good, but a few small nits found.

---

Reviewed 1 of 1 files at r1, 1 of 1 files at r2, 13 of 13 files at r3.
Review status: all files reviewed at latest revision, 2 unresolved discussions, all commit checks successful.

---

<sup>**[components/script/document_loader.rs, line 63 \[r3\]](https://reviewable.io:443/reviews/servo/servo/7499#-Jy9UR_zFjQEJEQWfFEP)** ([raw file](https://github.com/servo/servo/blob/d5454f6b90a19ff9548ff17b51b287c945ec08e0/components/script/document_loader.rs#L63)):</sup>
I think we don't want trailing comma in function declarations.

---

<sup>**[python/tidy.py, line 202 \[r2\]](https://reviewable.io:443/reviews/servo/servo/7499#-Jy9U8OnKEB5FfFKdqHd)** ([raw file](https://github.com/servo/servo/blob/2dfd438f3752bdf4ff3f73ee8d73c70a02845402/python/tidy.py#L202)):</sup>
Nit: no need for brackets here.

---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/servo/servo/7499)
<!-- Sent from Reviewable.io -->

metajack commented:
@jdm agreed

nerith committed:
python/tidy.py: Ignore macros in style nit check

Lines with macros were flagged as errors when checking
for space after a comma.

nerith committed:
Fix reported test-tidy errors

This fixes lines that were reported to have missing
space after a comma.

metajack commented:
@bors-servo r+

---

Reviewed 14 of 14 files at r4, 13 of 13 files at r5.
Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/servo/servo/7499)
<!-- Sent from Reviewable.io -->

bors-servo commented:
:pushpin: Commit d61a6e2 has been approved by `metajack`

<!-- @bors-servo r=metajack d61a6e2161bbce5182a5e1b9c504c55d5b2aa2f4 -->

bors-servo commented:
:hourglass: Testing commit d61a6e2 with merge 3424e23...

bors-servo commented:
:sunny: Test successful - [android](http://build.servo.org/builders/android/builds/2988), [gonk](http://build.servo.org/builders/gonk/builds/2147), [linux-dev](http://build.servo.org/builders/linux-dev/builds/73), [linux-rel](http://build.servo.org/builders/linux-rel/builds/70), [mac-dev-ref-unit](http://build.servo.org/builders/mac-dev-ref-unit/builds/73), [mac-rel-css](http://build.servo.org/builders/mac-rel-css/builds/72), [mac-rel-wpt](http://build.servo.org/builders/mac-rel-wpt/builds/73)
